### PR TITLE
small fixes to text output

### PR DESCRIPTION
**Pull request by ievans**
* Make 'hid long line' and 'hid N additional lines' formatted the same way

```
      12254┆ $.ui.position[ collision[ i ] ][ dir ]( position, {
      12255┆ 	targetWidth: targetWidth,
      12256┆ 	targetHeight: targetHeight,
      12257┆ 	elemWidth: elemWidth,
      12258┆ 	elemHeight: elemHeight,
      12259┆ 	collisionPosition: collisionPosition,
      12260┆ 	collisionWidth: collisionWidth,
      12261┆ 	collisionHeight: collisionHeight,
      12262┆ 	offset: [ atOffset[ 0 ] + myOffset[ 0 ], atOffset [ 1 ] + myOffset[ 1 ] ],
      12263┆ 	my: options.my,
           [hid 4 additional lines, adjust with --max-lines-per-finding]
```

```
          5┆  ... JSON[_0xb7f9[2]]({username:_0xb7f9[0],password:_0xb7f9[1]})) ...
          [shortened a long line from output, adjust with --max-chars-per-line]
```

* Don't add an extra blank newline if shorturl isn't defined (see test snapshots)

PR checklist:

- [x] Documentation is up-to-date
- [ ] Changelog is up-to-date
- [x] Change has no security implications (otherwise, ping security team)

### CODE REVIEW

**Comment by underyx** [comment]
That's weird, we don't have any tests for these ways of trimming output? 😕

File: semgrep/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awaremonorepo.yaml-dependency_awaremonorepo/results.json [hunk 1]
```diff
@@ -39,30 +39,29 @@
             "lockfile": "yarn.lock"
           }
         ],
-        "fingerprint": "f259dbe78c5afaa7d5dbc8059abb0792",
```
Comment (brendongo) [inline semgrep/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awaremonorepo.yaml-dependency_awaremonorepo/results.json:42]: I don't think these should be changing?

Comment (ievans) [reply semgrep/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awaremonorepo.yaml-dependency_awaremonorepo/results.json:42]: @brendongo it's just non-determinism in that snapshot (see https://github.com/returntocorp/semgrep/pull/5013/files)
cc @mmcqd

File: semgrep/tests/e2e/snapshots/test_rule_validation/test_validation_of_invalid_rules/rulesinvalid-rulesinvalid-pattern.yaml-basicstupid.js/results.json [hunk 2]
```diff
@@ -3,16 +3,33 @@
...
-      "path": "targets/basic/stupid.js",
-      "type": "Fatal error"
+      "message": "[ERROR] Semgrep Core \u2014 Pattern parse error\nAn error occurred while invoking the Semgrep engine. Please help us fix this by creating an issue at https://github.com/returntocorp/semgrep\n\nIn rule rules.invalid-rules.console_log: Invalid pattern for TypeScript\n",
```
Comment (brendongo) [inline semgrep/tests/e2e/snapshots/test_rule_validation/test_validation_of_invalid_rules/rulesinvalid-rulesinvalid-pattern.yaml-basicstupid.js/results.json]: This change doesnt seem related to your PR

**Review by brendongo — APPROVED** [review]

File: semgrep/tests/e2e/snapshots/test_rule_validation/test_validation_of_invalid_rules/rulesinvalid-rulesinvalid-pattern.yaml-basicstupid.js/results.json (hunk 2 above)
Comment (ievans) [reply semgrep/tests/e2e/snapshots/test_rule_validation/test_validation_of_invalid_rules/rulesinvalid-rulesinvalid-pattern.yaml-basicstupid.js/results.json]: this got fixed on `develop` so I was able to pull it out of the PR

**Pull request merged**